aiden_app/tools/utils/wtj_scraper.py

```diff
@@ -89,24 +131,29 @@ def __init__(self):
             "apiKey": script_dict["HERE_API_KEY"],
             "lang": "fr",
         }
+        logger.info("succesfully initialized WTJ scraper")
 
-    def _get_algolia_params(self, search_query: str, latlng: str):
-        params = {"hitsPerPage": 300, "query": search_query, "aroundLatLng": latlng, "aroundRadius": 2000000}
+    def _get_algolia_params(self, search_query: str, latlng: str, page: int):
+        params = {"hitsPerPage": 1000, "query": search_query, "aroundLatLng": latlng, "aroundRadius": 200000000000000, "page": page}
```

Comment:
Are the results ranked by location ? Does the location even matter with such a large radius?